### PR TITLE
Small docs fixes

### PR DESCRIPTION
**Pull request by carlbordum**
Hello, I have simply fixed a bunch of minor documentation issues

### CODE REVIEW

File: docs/general/schema-basics.md [hunk 1]
```diff
@@ -76,17 +74,6 @@ class Author:
...
-The `__future__` import allows us to annotate types that are defined later in
-the file, as per [PEP 563](https://www.python.org/dev/peps/pep-0563/). If you
-can’t use this feature (because your other code is incompatible with it), you
-need to quote the `Author` annotation.
-
-```python
-    ...
-    author: 'Author'
-    ...
-```
```
Comment (patrick91) [inline docs/general/schema-basics.md:88]: what do you think of this simplification @BryceBeagle @jkimbo ?

File: docs/types/object-types.md [hunk 2]
```diff
@@ -21,7 +21,8 @@ While reading about GraphQL you might have encountered 3 special object types: `
 - `Mutation` is the entry point for all the mutations
 - `Subscription` is the entry point for all the subscriptions.
 
-. TODO: add a link to docs on defining a schema.
+For a walk-through on how to define schemas, read the [schema
+basics](/docs/general/schema-basics.md).
```
Comment (patrick91) [inline docs/types/object-types.md:25]: 👍

File: docs/types/schema.md [hunk 3]
```diff
@@ -140,7 +140,7 @@ The name of the operation you want to execute, useful when sending a document wi
 Executes a GraphQL operation against a schema
 
 ```python
-async def execute_sync(query, variable_values, context_value, root_value, operation_name)`
+def execute_sync(query, variable_values, context_value, root_value, operation_name)`
```
Comment (patrick91) [inline docs/types/schema.md:143]: 💯 good spot

**Comment by patrick91** [comment]
@carlbordum thanks for this PR, I'll just check with the team if we are happy with the simplification. I think it might be ok to remove the `__future__` annotations for now, especially since there's (potentially) going to be changes to that, but we should explain why we use quotes around the type name :)

**Comment by carlbordum** [comment]
> @carlbordum thanks for this PR, I'll just check with the team if we are happy with the simplification. I think it might be ok to remove the `__future__` annotations for now, especially since there's (potentially) going to be changes to that, but we should explain why we use quotes around the type name :)

Maybe that can wait for https://strawberry.rocks/docs/concepts/typings?

**Comment by patrick91** [comment]
> > @carlbordum thanks for this PR, I'll just check with the team if we are happy with the simplification. I think it might be ok to remove the `__future__` annotations for now, especially since there's (potentially) going to be changes to that, but we should explain why we use quotes around the type name :)
> 
> Maybe that can wait for https://strawberry.rocks/docs/concepts/typings?

Good, point, maybe we can add a link to that when we fill the page :)

**Review by patrick91 — APPROVED** [review]
Thanks for doing this 😊

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Pull request merged**